contrib/README.md

```diff
@@ -0,0 +1,344 @@
+# Event Gateway on Kubernetes 
+
+This chart deploys the Event Gateway with etcd onto a Kubernetes cluster. Please note, the default instructions expect
+an existing kubernetes cluster that supports ingress, such as GKE. If your environment doesn't have ingress support
+set up, please follow the [minikube](MINIKUBE.md) instructions to set this up for your development environment.  
+
+## Contents
+
+1. [Quickstart](#quickstart)
+1. [Examples](#examples)
+    1. [Register a function](#register-a-function)
+    1. [Query all function](#query-all-function)
+    1. [Register an event](#register-an-event)
+    1. [Query all events](#query-all-events)
+    1. [Register a subscription](#register-a-subscription)
+    1. [Query all subscriptions](#query-all-subscriptions)
+    1. [Trigger an event](#trigger-an-event)
+1. [Configuration](#configuration)
+1. [Cleanup](#cleanup)
+
+## Quickstart
+
+Make sure you have helm installed on your machine and run `helm init` on your k8s cluster. This will set up the
+`helm` and `tiller` functions required for easy deployment of config files to your cluster. You can follow
+instructions [here](https://docs.helm.sh/using_helm/#quickstart) if you have not set this up previously.
+
+**NOTE:** This portion of the config expects you to have a pre-existing kubernetes cluster (not minikube). For 
+local development please check the [minikube](MINIKUBE.md) information.
+
+Once installed, navigate to the `event-gateway/contrib/helm` folder and install the following components:
+
+**etcd-operator**
+```bash
+helm install stable/etcd-operator --name ego [--namespace <namespace>]
+```
+
+**event-gateway**
+```bash
+helm install event-gateway --name eg [--namespace <namespace>]
+```
+
+This will install each of the `etcd-operator` and `event-gateway` into the `default` namespace in kubernetes. Please note,
+this namespace has no bearing on your Event Gateway `spaces` as outlined in the [docs](https://github.com/serverless/event-gateway/blob/master/README.md). If you'd like
+to install `etcd-operator` and `event-gateway` in another namespace, add the `--namespace <namespace>` option to
+ both `helm install` commands above.
+
+Next we'll need to collect the Event Gateway IP to use on the CLI. We have a couple of options available to reference the
+internal services of the kubernetes cluster exposed via Ingress:
+
+1. add Ingress IP to /etc/hosts (recommended)
```

Comment:
The switching between `eventgateway.minikube` and `EVENT_GATEWAY_URL` environment variable is confusing to me. I'd suggest consolidating on one or the other just to make it easier. The more forks in a tutorial, the more confusing it gets.

My guess would be to use the environment variable as some people may be averse to messing with their `/etc/hosts` file but will defer to you.